script/deploy/DeployL1RngAuction.s.sol

```diff
@@ -40,7 +41,8 @@ contract DeployL1RngAuction is Helpers {
         );
 
         new ChainlinkVRFV2DirectRngAuctionHelper(chainlinkRng, IRngAuction(address(rngAuction)));
-        new RngAuctionRelayerRemoteOwner(rngAuction);
+        new RngAuctionRelayerRemoteOwnerArbitrum(rngAuction);
+        new RngAuctionRelayerRemoteOwnerOptimism(rngAuction);
```

Comment:
I'm thinking we should move each one of these to a separate script since we don't need to redeploy the RngAuction or the other relays every time.

We could separate them like so:
- DeployL1RngAuction.s.sol
- DeployL1RelayOptimism.s.sol
- DeployL1RelayArbitrum.s.sol